hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -263,8 +263,8 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
         throw e;
 
       } catch (Throwable e) {
-        if (e.getMessage()
-            .contains(
+        if (e.getMessage() != null
```

Comment:
It'd better to create a util method for this pattern.

Reply:
Where would we use this new method?
If we would use it only in this class, and only in 2 places, then I think it is better not to "hide" what we do behind a method somewhere else in the code.

Reply:
If we had the method in the first place, I don't think we would miss the null check here. With this method, we can ensure the null check will be applied in any other places.